ansible/cluster.yml

```diff
@@ -53,3 +94,18 @@
     - node
   tags:
     - nodes
+
+- hosts:
+    - masters[0]
+  sudo: yes
+  roles:
+    - { role: opencontrail-master-provision, when: networking == 'opencontrail' }
+  tags:
+    - opencontrail-provision
+
+- hosts: nodes
+  sudo: yes
+  roles:
+    - { role: opencontrail-node-provision, when: networking == 'opencontrail' }
+  tags:
+    - opencontrail-provision
```

Comment:
There is an ordering issue that i'm not sure how to solve:
- Some opencontrail node components (e.g. the kubelet plugin) must be installed before the kubelet is started on the nodes;
- These provisioning tasks must execute after the master kubelet is running.

The kubelet on the nodes, to have the "opencontrail" plugin option requires the executable that is going to be called to exist. Ordering is something i really had trouble with. I would appreciate suggestions and/or better ways to do this. I agree that having so many top level roles is undesirable.